tests/components/Layout.test.js

```diff
@@ -1,7 +1,10 @@
 import { render } from '@testing-library/react'
 import TemplateWrapper from '@/components/Layout'
+import * as nextRouter from 'next/router'
 
 describe('<Layout />', () => {
+  nextRouter.useRouter = jest.fn() // eslint-disable-line
```

Comment:
Please use jest's module mocks for this.

Reply:
I will make sure I read into the Jest Module for updating tests in the future.